pages/devs/get-started/managing-gas.mdx

```diff
@@ -2,29 +2,121 @@ import { Callout } from 'nextra/components'
+## Overview
+
+**Invoking transactions causes network validators to do computations on your behalf and update the chain's state.** These actions are compensated via _gas_. Gas is paid by wallets who send transactions to the Allora chain.
+
+### What Changed in v0.7.0?
+
+**In its [v0.7.0 release](https://github.com/allora-network/allora-chain/releases/tag/v0.7.0), Allora incorporated the [x/feemarket module](https://github.com/skip-mev/feemarket), which means that gas calculations follow an [EIP-1559-like schedule](https://help.coinbase.com/en/coinbase/getting-started/crypto-education/eip-1559) (see: the [original EIP-1559](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1559.md)).** This requires transactions to be structured differently from other cosmos chains.
+
+### Why the Change Matters
+
+The EIP-1559-like system provides:
+- **Predictable fees**: More stable gas pricing mechanisms
+- **Better user experience**: Improved fee estimation and transaction processing
+- **Network efficiency**: Optimized resource allocation for validators
+- **Ethereum compatibility**: Familiar fee structure for developers
+
+## Transaction Structure Evolution
+
+### Before v0.7.0 (Legacy Format)
 
 Prior to `allora-chain` `v0.7.0` and in many other Cosmos chains, transactions are typically structured like so:
 
 ```
 allorad tx emissions ... --from ACCOUNT_NAME --node RPC --chain-id allora-testnet-1 --keyring-backend test --keyring-dir ~/.allorad/ --gas auto --gas-adjustment 1.2 --fees 2024700uallo ...
 ```
 
+**Legacy Format Characteristics**:
+- Used `--gas auto` for automatic gas estimation
+- Used `--fees Xuallo` format for fee specification
+- Relied on simple fee calculation methods
+
+### Current Format (v0.7.0+)
+
 Since `v0.7.0`, transactions should instead abide by the structure:
+
 ```
 allorad tx emissions ... --from ACCOUNT_NAME --node RPC --chain-id allora-testnet-1 --keyring-backend test --keyring-dir ~/.allorad/ --gas 130206 --gas-adjustment 1.2 --gas-prices 10uallo ...
 ```
 
-To emphasize: This^^ is the way transactions should be structured today using `allorad`.
+**To emphasize: This^^ is the way transactions should be structured today using `allorad`.**
+
+**Current Format Benefits**:
+- More precise gas control with explicit gas limits
+- Standardized gas pricing across the network
+- Better compatibility with fee market mechanisms
+
+## Key Parameter Changes
 
-The specific differences are:
+### Fee Structure Update
 
-__`fees Xuallo` becomes `gas-prices 10uallo`__
+**`fees Xuallo` becomes `gas-prices 10uallo`**
```

Comment:
The way we specify gas related options actually didn't change since the integration of `x/feemarket`, this module only brings a variable gas price computed on-chain, the current gas price can be queried with `allorad q feemarket gas-prices`.

Even with this EIP-1559 implementation we can specify `--gas auto` which will try to estimate gas usage, or use `--fees Xuallo` to set a limit to the gas fees to spent on a tx.